### ISSUE TITLE
Address change doesn't get synced/visible in Simple Contacts

### ISSUE THREAD

**Issue by keunes** [SimpleMobileTools/Simple-Contacts#678]
Hello,

First of all: thanks a lot for working on these apps and keeping them open source!

I added today a couple of addresses in the Simple Contacts app, which (via DavX5) are synced to my Nextcloud. There were some inconsistencies (like, for some addresses the country was in full caps), which I corrected in Nextcloud. However, these changes don't seem to appear in Simple Contacts, despite contact syncing working in general (to test it, I created a new contact, synced DavX5 on my phone, and confirmed that the new contact appeared).

Not sure why this is happening. Might it be related to #364?

**Comment by tibbi** [comment]
what app version are you using?

**Comment by keunes** [comment]
v 6.14.2 Pro (from F-Droid)

**Comment by tibbi** [comment]
well ye, Simple Contacts handles the address as 1 value, it isnt split. So this is a duplicate of #364

**Comment by keunes** [comment]
Ok, giving a thumbs-up there. But shouldn't the address then be updated to only include the Street + House number, in accordance with the latest data received from the server?

**Comment by tibbi** [comment]
Im not tweaking that data in any way. What is stored under the Address field of the contact, that is shown in Simple Contacts.

**Comment by keunes** [comment]
Not saying you're tweaking the data. But
1. I added this data in Simple Contacts
2. It got synced to Nextcloud via DavX5
3. I removed post code, city & country from the 'Address' field and put it into separate fields
4. After syncing, I would expect Simple Contacts to display only the street + house number, because that's the only info now stored in the Address field of the contact (instead, it still shows the full address).
